src/fides/lib/oauth/oauth_util.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import json
+from datetime import datetime
+
+from fastapi.security import SecurityScopes
+from jose import exceptions, jwe
+from sqlalchemy.orm import Session
+
+from fides.ctl.core.config import FidesConfig
+from fides.lib.cryptography.schemas.jwt import (
+    JWE_ISSUED_AT,
+    JWE_PAYLOAD_CLIENT_ID,
+    JWE_PAYLOAD_SCOPES,
+)
+from fides.lib.exceptions import AuthorizationError
+from fides.lib.models.client import ClientDetail
+
+
+def extract_payload(jwe_string: str, encryption_key: str) -> str:
+    """Given a jwe, extracts the payload and returns it in string form."""
+    return jwe.decrypt(jwe_string, encryption_key)
+
+
+def is_token_expired(issued_at: datetime | None, token_duration_min: int) -> bool:
+    """Returns True if the datetime is earlier than token_duration_min ago."""
+    if not issued_at:
+        return True
+
+    return (datetime.now() - issued_at).total_seconds() / 60.0 > token_duration_min
+
+
+def verify_oauth_client(
+    security_scopes: SecurityScopes,
+    authorization: str,
+    *,
+    db: Session,
+    config: FidesConfig,
+) -> ClientDetail:
```

Comment:
Very similar code exists in src/fides/api/ops/util/oauth_util.py and I think that is the version we should be using, not this one.  Ops was overriding the fideslib version.